engines/ultima/ultima8/world/actors/avatar_mover_process.h

```diff
@@ -89,6 +93,10 @@ class AvatarMoverProcess : public Process {
 	bool _tryTurnRight;
 	bool _tryMoveForward;
 	bool _tryMoveBack;
+	bool _tryMoveLeft;
+	bool _tryMoveRight;
+	bool _tryMoveUp;
+	bool _tryMoveDown;
```

Comment:
There's starting to be quite a few of these. Should these be switched flags for a movement state variable?

Reply:
Yeah it might be nicer - that would also avoid impossible combinations (like trymoveup + trymovedown at the same time).  Doesn't have to be part of this change.